examples/suit_update/README.md

```diff
@@ -0,0 +1,558 @@
+# Overview
+
+This example shows how to integrate SUIT-compliant firmware updates into a
+RIOT application. It implements basic support of the SUIT architecture using
+the manifest format specified in
+[draft-moran-suit-manifest-04](https://datatracker.ietf.org/doc/draft-moran-suit-manifest/04/).
+
+Table of contents:
+
+- [Prerequisites][prerequisites]
+- [Setup][setup]
+  - [Signing key management][key-management]
+  - [Setup a wired device using ethos][setup-wired]
+    - [Provision the device][setup-wired-provision]
+    - [Configure the network][setup-wired-network]
+  - [Alternative: Setup a wireless device behind a border router][setup-wireless]
+    - [Provision the wireless device][setup-wireless-provision]
+    - [Configure the wireless network][setup-wireless-network]
+  - [Start aiocoap fileserver][start-aiocoap-fileserver]
+- [Perform an update][update]
+  - [Build and publish the firmware update][update-build-publish]
+  - [Notify an update to the device][update-notify]
+- [Detailed explanation][detailed-explanation]
+- [Automatic test][test]
+
+## Prerequisites
+[prerequisites]: #Prerequisites
+
+- Install python dependencies (only Python3.6 and later is supported):
+
+      $ pip3 install --user ed25519 pyasn1 cbor click
+
+- Install aiocoap from the source
+
+      $ pip3 install --user --upgrade "git+https://github.com/chrysn/aiocoap#egg=aiocoap[all]"
+
+  See the [aiocoap installation instructions](https://aiocoap.readthedocs.io/en/latest/installation.html)
+  for more details.
+
+- Install aiocoap-fileserver
```

Comment:
This step is not needed anymore when installing aiocoap from source.

Reply:
fixed